src/utils.js

```diff
@@ -180,6 +181,64 @@ export function buildLocaleDateAndTime(dateTime, timezone, locale) {
     .format('L LT');
 }
 
+export function getSelectedSlipData(pickSlipsData, selectedRequestId) {
+  const sel = pickSlipsData.filter((pickSlip) => {
+    return pickSlip['request.requestID'] === selectedRequestId;
+  })[0];
+  if (sel === undefined) {
+    return [];
+  } else {
+    return [sel].flat();
+  }
+}
+
+export function getSelectedSlipDataMulti(pickSlipsData, selectedRows) {
+  const sel = pickSlipsData.filter((pickSlip) => {
+    return Object.keys(selectedRows).includes(pickSlip['request.requestID']);
+  });
+  if (sel === undefined) {
+    return [];
+  } else {
+    return [sel].flat();
+  }
+}
+
+export function selectedRowsNonPrintable(pickSlipsData, selectedRows) {
+  if (!size(selectedRows)) {
+    return true;
+  }
+  const sel = pickSlipsData.filter((pickSlip) => {
+    return Object.keys(selectedRows).includes(pickSlip['request.requestID']);
+  });
+  if (sel === undefined) {
+    return true;
+  } else return sel.length === 0;
+}
+
+export function isPrintable(requestId, pickSlips) {
+  let matched;
+  if (pickSlips !== undefined) {
+    matched = pickSlips.filter((pickSlip) => {
+      return pickSlip.request.requestID === requestId;
+    })[0];
+  }
+  return matched != null;
+}
```

Comment:
Is it expected behaviour when `pickSlips` is `undefined`? In this case we will have the following comparison 
`undefined != null`.

Reply:
do you mean `pickSlip !=null` ?

I don't understand what you mean by `undefined != null`